lib/std/Thread.zig

```diff
@@ -38,6 +38,180 @@ else
 
 impl: Impl,
 
+pub const maxNameLen = switch (std.Target.current.os.tag) {
```

Comment:
This is public so let's make it conform to the [style guide](https://ziglang.org/documentation/master/#Style-Guide):

```suggestion
pub const max_name_len = switch (std.Target.current.os.tag) {
```